dashboard/app/mailers/ops_mailer.rb

```diff
@@ -34,9 +34,9 @@ def workshop_reminder(workshop, recipient)
     @workshop = workshop
     @recipient = recipient
 
-    subject = "Important: Your #{@workshop.phase_info[:long_name]} workshop is coming up in
```

Comment:
the real fix should be to make phase a required field and restrict its values so that phase_info is never nil. There is a generalized safe lookup method already (but I think it looks ugly with []), do you think it would be better to use try?